install/lib/class.installer.php

```diff
@@ -194,7 +194,7 @@ private function checkRequirements()
         $errors = [];
 
         // Check for PHP 5.6+
-        if (version_compare(phpversion(), '5.6', '<=')) {
+        if (\Composer\Semver\Comparator::lessThan(phpversion(), '5.6')) {
             $errors[] = array(
                 'msg' => __('PHP Version is not correct'),
                 'details' => __('Symphony requires %1$s or greater to work, however version %2$s was detected.', array('<code><abbr title="PHP: Hypertext Pre-processor">PHP</abbr> 5.3</code>', '<code>' . phpversion() . '</code>'))
```

Comment:
The text will say that you need at least php 5.3.

Reply:
Good catch!